doc/hands_on/books/to_parquet.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+from pyspark.sql import SparkSession
+
+# This script converts the JSON files into a parquet file
+
+spark = SparkSession \
+    .builder \
+    .appName("Converter") \
+    .getOrCreate()
+
+json_df = spark.read.json("./book-*.json", multiLine=True)
+json_df.printSchema()
+json_df.coalesce(1).write.parquet("output.parquet")
```

Comment:
```suggestion
json_df.coalesce(1).write.parquet("books.parquet")
```

Reply:
For some reasons it creates a folder (called `output.parquet`) that contains the one part file.
```suggestion
json_df.coalesce(1).write.parquet("output")
```

Reply:
Aha yes, that is correct, it writes to folder.

Reply:
Please do not forget to commit your suggestion.